tests/terraform/checks/resource/aws/example_DBInstanceMinorUpgrade/main.tf

```diff
@@ -0,0 +1,52 @@
+resource "aws_rds_cluster_instance" "fail" {
+  name                         = "bar"
+  performance_insights_enabled = false
+  publicly_accessible = true
+  auto_minor_version_upgrade = false
+  tags                = { test = "Fail" }
+}
+
+resource "aws_rds_cluster_instance" "fail2" {
+  name                         = "bar"
+  performance_insights_enabled = false
+  publicly_accessible = true
+  auto_minor_version_upgrade = false
+  tags                = { test = "Fail" }
+}
```

Comment:
you probably meant to remove the property `auto_minor_version_upgrade` for one of the resources 🙂

Reply:
indeed  i did